recipes/knoten/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "knoten" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/knoten/archive/{{ version }}.tar.gz
+  sha256: d671f1c8c32107cf1ef1d2dadf236ba98f9f19dcfb744e302cb20860748164b0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win or py2k ]
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python >=3.7,<3.8
+    - setuptools 
+    - pip 
+  run:
+    - ale >=0.3.2
+    - python >=3.7,<3.8 
+    - csmapi >=1.0
+    - gdal >=3.0.0
+    - jinja2
+    - {{ pin_compatible('numpy') }}
```

Comment:
Just use `numpy`.  `pin_compatible` is only needed when numpy is used to compile the package.